components/Locale/basic_usage.rst

```diff
@@ -0,0 +1,49 @@
+Basic Usage
+===========
+
+LocaleContext
+-------------
+
+The **LocaleContext** allows you to manage the currently used locale.
+
+.. code-block:: php
+
+	<?php
+
+	use Sylius\Component\Locale\Context\LocaleContext;
+
+	$storage; //Your's storage
```

Comment:
Space after ``//``, Your storage.

But we should create some basic storage here. Or maybe introduce ``InMemoryStorage`` storage or smth, hm? Just for the sake of examples + tests. WDYT?